### PR TITLE
working prettier

### PR DESCRIPTION
**Pull request by barbaramistrz**
Added prettier to format all files consistently - https://prettier.io/

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -22,23 +24,25 @@
...
- background-color:   lightgray
-;
-">
-<!-- #FDB3BC #F27881 #FFF1F5 #FBDFE7 #EF5B67 #FBBBC1 -->
-  <noscript>You need to enable JavaScript to run this app.</noscript>
-  <div id="root" style="height: 100vh"></div>
-  <!--
+  <body
+    id="body"
+    style="
```
Comment (szynszyliszys) [inline public/index.html:32]: Huh, why is the styling even here inlined? I guess you could make a card to move this to the stylesheet. Also have image in the assets folder downloaded, as you never know when unsplash link can change :)

File: src/FavouriteRecipes.js [hunk 2]
```diff
@@ -2,12 +2,11 @@ import React from "react";
...
 const FavouriteRecipes = () => {
-  if (JSON.parse(localStorage.getItem("favourites")) === null){
-    localStorage.setItem('favourites', JSON.stringify([]));
-
+  if (JSON.parse(localStorage.getItem("favourites")) === null) {
+    localStorage.setItem("favourites", JSON.stringify([]));
   }
   const recipes = JSON.parse(localStorage.getItem("favourites"));
-  console.log(recipes)
+  console.log(recipes);
```
Comment (szynszyliszys) [inline src/FavouriteRecipes.js]: Not related to this PR, but I noticed it now: all these console.log() lines should never make it to the production repo. They are great for debugging, but when submitting PR with code to be tested, they should all be gone. Could you create a card for that?

**Review by szynszyliszys — CHANGES_REQUESTED** [review]
It looks so much better, great job! :)
I added two comments that are not connected to the PR itself, but would be good to create a followup tickets and tackle this later. There are things that would be great to change in the scope of this PR:

1. Add ticket link in the description so it is easy to go to ticket and check what it was about
2. Add `.prettierrc` file in the main directory. For now prettier is formatting files with your local settings, but if someone else comes to the project, the files could get reformatted with their settings if they are different.
Example `.prettierrc` file content from my project:

```
{
  "$schema": "http://json.schemastore.org/prettierrc",
  "singleQuote": true,
  "trailingComma": "es5",
  "printWidth": 100
}
```

File: public/index.html (hunk 1 above)
Comment (barbaramistrz) [reply public/index.html:32]: good points, added these to the backlog!

File: src/.prettierrc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (szynszyliszys) [inline src/.prettierrc:1]: Can you please fix indentation in this file and add newline at the end? I suggest installing `final-newline` extension in VSCode to help out with that

**Review by szynszyliszys — APPROVED** [review]

**Pull request merged**